pkgs/top-level/all-packages.nix

```diff
@@ -1172,15 +1172,28 @@ let
 
   cron = callPackage ../tools/system/cron { };
 
-  cudatoolkit5 = callPackage ../development/compilers/cudatoolkit/5.5.nix {
+  cudatoolkit5 = callPackage ../development/compilers/cudatoolkit/generic.nix {
+    version = "5.5.22";
```

Comment:
Sure no problem. Just curious, but why the extra files? I noticed usage of both conventions throughout nixpkgs.